### ISSUE TITLE
rm can't remove broken symlink

### ISSUE THREAD

**Issue by roosephu** [nushell/nushell#5241]
### Describe the bug

rm can't remove broken symlink. An error is thrown:
```
Error:
  × No valid paths
   ╭─[entry #21:1:1]
 1 │ rm aaa
   · ─┬
   ·  ╰── no valid paths
   ╰────
```

### How to reproduce

Assuming the file `nonexistence` doesn't exist. 
```bash
ln -s nonexistent aaa
rm aaa
```

### Expected behavior

The broken symlink `aaa` should be removed. 

### Screenshots

<img width="211" alt="image" src="https://user-images.githubusercontent.com/3055937/164041744-445e057b-464f-4a8f-b24b-2dbebdcb685f.png">

### Configuration

| key                | value                                 |
| ------------------ | ------------------------------------- |
| version            | 0.61.0                                |
| branch             |                                       |
| tag                |                                       |
| build_os           | macos-aarch64                         |
| rust_version       | rustc 1.59.0                          |
| cargo_version      | cargo 1.59.0                          |
| pkg_version        | 0.61.0                                |
| build_time         | 2022-04-12 20:38:15 +00:00            |
| build_rust_channel | release                               |
| features           | dataframe, default, trash, which, zip |
| installed_plugins  |                                       |

### Additional context

Tried `rm --force aaa`. No error but the symlink `aaa` still exists.

**Comment by kubouch** [comment]
How did you solve this? I have the same issue.

**Comment by roosephu** [comment]
I didn't solve it. I closed it in favor of #3977.
